twindb_backup/backup.py

```diff
@@ -43,10 +43,10 @@ def backup_mysql(run_type, config):
 
     try:
         if not config.getboolean('source', 'backup_mysql'):
-            log.debug('Not backing up MySQL')
-            return
+            raise TwinDBBackupError('MySQL backups are not enabled in config')
```

Comment:
Why do we treat it differently as compared to what we are doing in the exception handling below. When there is no option defined we simply ignore it. However, if the config value is set to false then we throw an error. This can be confusing.